src/sc/src/controllers/partitions/controller.rs

```diff
@@ -70,6 +70,7 @@ impl PartitionController {
         // info!("spu controller is terminated");
     }
 
+    #[instrument(skip(self, listener), name = "partition_controller_sync")]
```

Comment:
why renaming?

Reply:
I was trying to make a conventional naming scheme for all of the controllers. If you don't like it I can remove the rename